app/views/admin/creator_settings/_form.html.erb

```diff
@@ -78,3 +78,21 @@
     </div>
   </fieldset>
 </div>
+
+<div class="crayons-field mt-6 align-left">
+  <fieldset aria-describedby="section-description">
+    <legend class="crayons-field__label mb-2">Finally, please agree to the following:</legend>
+    <div>
+      <div class="mb-2">
+        <%= hidden_field_tag :checked_code_of_conduct, "0" %>
+        <%= check_box_tag :checked_code_of_conduct, "1", false, class: "crayons-checkbox", required: true %>
+        <label for="checked_code_of_conduct">I agree to uphold our <a href="/code-of-conduct">Code of Conduct</a>.</label>
+      </div>
+      <div>
+        <%= hidden_field_tag :checked_terms_and_conditions, "0" %>
+        <%= check_box_tag :checked_terms_and_conditions, "1", false, class: "crayons-checkbox", required: true %>
+        <label for="checked_terms_and_conditions">I agree to our <a href="/terms-and-conditions">Terms and Conditions</a>.</label>
```

Comment:
There is a bit of a delay with the `required` field on both of these checkboxes. I see the "required" pop up as expected, but it doesn't appear as. quickly as the others on the form do. Additionally, @Ridhwana [stated](https://github.com/forem/forem/pull/15430#discussion_r753064997) that she was only able to see the "required" pop up upon hover.

@nickytonline do you have any suggestions for how to address this?

Reply:
Hmm, not sure at the moment. That's peculiar...